client/app/app.module.js

```diff
@@ -1305,6 +1305,7 @@ angular.injector(['ng'])
         // TODO: add error handler if incorrect data provided or missed
         ngModule
             .constant('API_URL', response.data['API_URL'] || '')
+            .constant('iam_API_URL', response.data['API_URL'].slice(0, -18))
```

Comment:
what is 18?

Reply:
"/reporting-service" in URL. Pathes of new service don't contains this part.

Reply:
Please, leave an appropriate comment there.